pyiron_atomistics/lammps/potentials.py

```diff
@@ -373,7 +386,12 @@ def view_potentials(self):
     @property
     def df(self):
         if self._df is None:
-            df = self._df_candidates.iloc[0]
+            if self._choice > len(self._df_candidates):
+                raise ValueError(
+                    f"Cannot choose {self._choice} among {len(self._df_candidates)} "
+                    f"available choices."
+                )
+            df = self._df_candidates.iloc[self._choice]
             if len(self._df_candidates) > 1:
                 warnings.warn(
                     f"Potential not uniquely specified - use default {df.Name}"
```

Comment:
In relation with my comment above, I would issue this warning only if `choice` is not explicitly set.